share/jupyterhub/templates/page.html

```diff
@@ -140,6 +140,16 @@ <h4 class="modal-title" id="{{key}}-label">{{title}}</h4>
   </nav>
 {% endblock %}
 
+
+{% block announcement %}
+{% if announcement %}
+<div class="container text-center">
```

Comment:
Can this get the class 'announcement', too, for easier CSS? Then I think this is good to go.